exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/store/ZookeeperPersistentStore.java

```diff
@@ -63,7 +63,18 @@ public PersistentStoreMode getMode() {
 
   @Override
   public V get(final String key) {
-    final byte[] bytes = client.get(key);
+    return get(key, null);
```

Comment:
Minor point... Should this version simply call client.get as before? Should the version that takes a version require the version? Would seem to keep things a bit simpler that way...